creature.py

```diff
@@ -1,13 +1,19 @@
 import random
+from weapon import *
+
 
 class Creature:
     id = 1
 
     def __init__(self):
-        self.health = 50
+        self.health = 25
         self.name = f"creature_{Creature.id}"
+        self.ac = 11
```

Comment:
What is `ac`? It's not good to give such names so it's not obvious what it means.